dumb-jump.el

```diff
@@ -1528,7 +1528,32 @@ or most optimal searcher."
     (:type "type" :supports ("ag" "grep" "rg" "git-grep") :language "apex"
            :regex "(class|interface)\\s*JJJ\\b"
            :tests ("class test:" "public class test implements Something")
-           :not ("class testnot:" "public class testnot implements Something")))
+           :not ("class testnot:" "public class testnot implements Something"))
+
+    ;; odin
+    (:type "function" :supports ("ag" "grep" "rg" "git-grep") :language "odin"
+           :regex "\\s*\\bJJJ\\s*::\\s*proc"
```

Comment:
There may be whitespaces between the 2 colons as well.

Odin procedure declarations may specify an explicit calling convention. See here for more information: https://odin-lang.org/docs/overview/#calling-conventions

Reply:
Oh you're right